packages/app-check/src/api.ts

```diff
@@ -73,6 +90,9 @@ export function initializeAppCheck(
     }
   }
 
+  // Only read global variable on first call to `initializeAppCheck()`.
+  initializeDebugMode();
```

Comment:
It should be called only once for all app instances, not per app instance.

Reply:
Changed.